managed/services/agents/mysql.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/percona/pmm/version"
 )
 
+var mysqlExporterVersionWithPluginCollector = version.MustParse("2.36.0")
```

Comment:
JFYI: to simplify testing you can use `2.36.0-0`, in that case all pre-release versions will match.